### DISCUSSION TITLE
C++ Api Stability Roadmap

### DISCUSSION THREAD

**Discussion by connoranderson** [foxglove/mcap#1101] [General] [answered]
Hi all, is there a projected date for C++ mcap api to reach v1.0?

**Comment by jtbandes** [comment, marked as answer]
Hi, we actually bumped the version to 1.0.0 last month: https://github.com/foxglove/mcap/pull/793

We have yet to update the version on the conan-center-index. I've created an internal ticket (FG-2167) tracking this task.
